bevy_proto_backend/src/proto/prototypes.rs

```diff
@@ -48,6 +48,25 @@ impl<'w, T: Prototypical> PrototypesMut<'w, T> {
         handle
     }
 
+    /// Load the prototypes at the given path.
+    ///
+    /// This will also store strong handles to the prototypes in order to keep them loaded.
+    ///
+    /// To load without automatically storing the handles, try using [`AssetServer::load_folder`].
+    pub fn load_folder<P: Into<Cow<'static, str>>>(&mut self, path: P) -> Result<Vec<Handle<T>>, AssetServerError> {
+        let path = path.into();
+        let handles = self.asset_server.load_folder(path.as_ref())?;
+        let handles: Vec<_> = handles.iter().map(|handle| handle.clone().typed::<T>()).collect();
+
+        handles.iter().for_each(|handle| {
+            let path = self.asset_server.get_handle_path(handle).unwrap().path().to_str().unwrap().to_string();
```

Comment:
This is a bit gross, should we handle the error cases instead of calling `unwrap` twice?

Reply:
Theoretically this should always return a path, right? If so, I think we can go ahead and just unwrap it (maybe with an `.expect()` instead).

Same for the call to `to_str` (which we could replace with `to_str_lossy` but that could break the mapping).

Reply:
I added some errors for this, let me know what you think